cpp/include/cudf/aggregation.hpp

```diff
@@ -120,7 +120,9 @@ class aggregation {
     TDIGEST,         ///< create a tdigest from a set of input values
     MERGE_TDIGEST,   ///< create a tdigest by merging multiple tdigests together
     HISTOGRAM,       ///< compute frequency of each element
-    MERGE_HISTOGRAM  ///< merge partial values of HISTOGRAM aggregation,
+    MERGE_HISTOGRAM, ///< merge partial values of HISTOGRAM aggregation,
+    MAX_BY,          ///< max reduction by another column
+    MIN_BY           ///< min reduction by another column
```

Comment:
Since we are specifying just one `value` column (which is a struct of 2 columns), this comment is not accurate. It should be s something like `retrieve value on a child column based on min/max value of another child column`.

Reply:
updated.